config/crd/bases/config.ratify.deislabs.io_verifiers.yaml

```diff
@@ -113,7 +113,9 @@ spec:
                 description: The type of artifact this verifier handles
                 type: string
               name:
-                description: Name of the verifier
+                description: |-
+                  TODO: update all docs spec to use type and add deprecation warning in spec to name field
```

Comment:
it's a bit weird to have a TODO comment in CRD, we can remove it. And add `Deprecated` tag.